h/h_api/bulk_api/entry_point.py

```diff
@@ -0,0 +1,91 @@
+"""Wrappers for common ways to use the bulk API."""
+
+import json
+from io import StringIO
+from json import JSONDecodeError
+
+from h.h_api.bulk_api.command_builder import CommandBuilder
+from h.h_api.bulk_api.command_processor import CommandProcessor
+from h.h_api.bulk_api.executor import AutomaticReportExecutor, Executor
+from h.h_api.bulk_api.observer import Observer, SerialisingObserver
+from h.h_api.exceptions import InvalidJSONError
+
+
+class BulkAPI:
+    """Convenience methods for streaming to and from the BulkAPI."""
+
+    @classmethod
+    def from_stream(cls, lines, executor, observer=None):
```

Comment:
When we find out if we are getting streams or strings we might be able to remove something here?